src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -570,14 +572,15 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
     timestamp?: number,
     onlyOneBlock: boolean = false
   ) => {
-    await this.#blockBeingSavedPromise;
```

Comment:
The miner has its own internal mechanism for making sure only one block can be mined at a time, so I don't think this is necessary. It didn't affect the tests, but this is the scariest part of the change for me :sweat_smile:

Reply:
:sweat_smile: